src/sidebar/helpers/annotation-sharing.ts

```diff
@@ -32,12 +27,11 @@ export function annotationSharingLink(annotation) {
  * Generate a URI for sharing: a bouncer link built to share annotations in
  * a specific group (groupID) on a specific document (documentURI). If the
  * `documentURI` provided is not a web-accessible URL, no link is generated.
- *
- * @param {string} documentURI
- * @param {string} groupID
- * @return {string|null}
  */
-export function pageSharingLink(documentURI, groupID) {
+export function pageSharingLink(
+  documentURI: string,
+  groupID: string
```

Comment:
Now would be a good time to rename this to `groupId` following our identifier naming conventions, which in turn generally follow Web/JS API conventions.